src/src/Commands/Group/GroupFetchCommand.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace QIT_CLI\Commands\Group;
+
+use QIT_CLI\TestGroup;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class GroupFetchCommand extends Command {
+
+	protected static $defaultName = 'group:fetch'; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.PropertyNotSnakeCase
+
+	/** @var TestGroup */
+	protected $test_group;
+
+	public function __construct( TestGroup $test_group ) {
+		$this->test_group = $test_group;
+		parent::__construct();
+	}
+
+	protected function configure() {
+		$this
+			->setDescription( 'Fetch a group of tests using a group identifier.' )
```

Comment:
The spacing on this seems a bit weird in GitHub but might be fine, just noting it!

Reply:
Looks good locally. Just a GH thing

Reply:
Ah cool, thanks for confirming!